include/nbl/asset/filters/dithering/CPrecomputedDither.h

```diff
@@ -77,15 +77,15 @@ namespace nbl
 								asset::encodePixels<decodeFormat>(outData + offset, decodeBuffer);
 							};
 
-							CBasicImageFilterCommon::executePerRegion(flattenDitheringImage.get(), decode, flattenDitheringImage->getRegions(chosenMipmap).begin(), flattenDitheringImage->getRegions(chosenMipmap).end());
+							CBasicImageFilterCommon::executePerRegion(flattenDitheringImage.get(), decode, flattenDitheringImage->getRegions(chosenMipmap).data(), flattenDitheringImage->getRegions(chosenMipmap).data() + flattenDitheringImage->getRegions(chosenMipmap).size());
```

Comment:
could you fix the double calls, and save the span into a temporary variable? Currentl:
1. its horrible line clutter
2. `getRegions` actually involves a binary search, so I'd rather not perform it twice

Reply:
resolved